services/QuillLMS/client/app/bundles/Teacher/containers/SocialStudiesSubjectPage.tsx

```diff
@@ -0,0 +1,196 @@
+import * as React from 'react';
+import ReactMarkdown from 'react-markdown';
+
+import QuestionsAndAnswers from './QuestionsAndAnswers'
+
+import PreviewCard from '../components/shared/preview_card.jsx';
+import { requestGet } from '../../../modules/request';
+
+const largeIconBaseSrc = `${process.env.CDN_URL}/images/icons/l`
```

Comment:
maybe we should add this to the Shared bundle and make this a pattern with other icon sizes?

Reply:
Good idea!